src/components/templates/Audience/Audience.tsx

```diff
@@ -340,32 +325,13 @@ export const Audience: React.FC<AudienceProps> = ({ venue }) => {
     // @debt This should probably be all rolled up into a single canonical component for emoji reactions/etc
     const renderReactionsContainer = () => (
       <>
-        <div className="emoji-container">
-          {burningReactions.map((reaction) => (
-            <button
-              key={reaction.name}
-              className="reaction"
-              onClick={() => reactionClicked(reaction.type)}
-              id={`send-reaction-${reaction.type}`}
-            >
-              <span role="img" aria-label={reaction.ariaLabel}>
-                {reaction.text}
-              </span>
-            </button>
-          ))}
-          <div
-            className="mute-button"
-            onClick={() => setIsAudioEffectDisabled((state) => !state)}
-          >
-            <FontAwesomeIcon
-              className="reaction"
-              icon={isAudioEffectDisabled ? faVolumeMute : faVolumeUp}
-            />
-          </div>
-          <button className="leave-seat-button" onClick={leaveSeat}>
-            Leave Seat
-          </button>
-        </div>
+        <ReactionsBar
+          reactions={burningReactions}
+          venueId={venueId}
+          leaveSeat={leaveSeat}
```

Comment:
This prop is a callback and should be `onLeaveSeat`. Think of the regular html `button` it's callback prop is `onChange` not `change`. Almost all of the props that accept functions should be like this. There are some rare cases ofc.

Reply:
This might actually need to be more specific and changed to `onClickLeaveSeat` to indicate that the function will be called when leave seat is clicked.

Reply:
fixed in 9cbbfe8f7c30d7ffb04d2dd93820d4d5add71ec4